05week/spaceTravelToMars.js

```diff
@@ -10,6 +10,42 @@ let jobTypes = {
 };
 
 // Your code here
+class CrewMember{
+  constructor(name,job,specialSkill){
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+    this.ship = null
+  }
+  enterShip(ship){
+    if(ship.missionStatement()){
+      ship.crew.push(this)
+      this.ship = ship;
+      return `${this.name} has a job of ${this.job} and is asigned to ${this.ship.name} and that ship
+      is a type ${this.ship.type}`
+    } else {
+      return
+    }
+  }
+}
+class Ship{
+  constructor(name,type,ability){
+    this.name = name;
+    this.type = type;
+    this.ability = ability
+    this.crew =[];
+  }
+  missionStatement(){
+    for(let i = 0; i<this.crew;i++){}
+    if(CrewMember.job === this.type){
```

Comment:
you cannot compare the job and type directly, you have to use the job types object at the beginning of this file to find the matches.